demo/Deployment.yml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1 # for versions before 1.8.0 use apps/v1beta1
+kind: Deployment
+metadata:
+  name: spring-boot-deployment
+spec:
+  selector:
+    matchLabels:
+      app: spring-boot-app
+  replicas: 3 # tells deployment to run 3 pods matching the template
+  template: # create pods using pod definition in this template
+    metadata:
+      labels:
+        app: spring-boot-app
+    spec:
+      containers:
+      - name: spring-boot-app-ui
+        image: <private docker repo>/ui:v1.2
+        ports:
+        - containerPort: 8081
```

Comment:
https://github.com/Microsoft/containers-rest-cosmos-appservice-java/blob/master/ui/Dockerfile says the uses 8080. Does this work?

Reply:
The image I built and deployed has the UI switched to 8081 for now. I couldn't get both the UI and API running on the same pod, since both containers wanted to expose 8080. Are there ways around this?

Reply:
Can you leave a note of the port change in this yaml? LGTM otherwise

Reply:
Actually gonna close this one and reopen a revision where we don't have to change the configuration of the spring-boot project.